sklearn/feature_extraction/image.py

```diff
@@ -501,30 +501,42 @@ def transform(self, X):
 
         Examples
         --------
-
+        >>> import scipy as sp
         >>> from sklearn.feature_extraction.image import PatchExtractor
-        >>> X = np.random.randint(10, size=(2, 2, 2))
-        >>> X
-        array([[[5, 0],
-                [9, 6]],
-
-              [[2, 0],
-                [5, 2]]])
 
-        >>> pe = PatchExtractor(patch_size=(2, 1))
-        >>> pe.fit(X)
-        >>> pe.transform(X)
-        array([[[ 5.],
-                [ 9.]],
+        >>> X = sp.misc.face()
+        >>> X.shape
+        (768, 1024, 3)
 
-              [[ 0.],
-                [ 6.]],
+        >>> X[0]
+        array([[121, 112, 131],
+               [138, 129, 148],
+               [153, 144, 165],
+               ...,
+               [119, 126,  74],
+               [131, 136,  82],
+               [139, 144,  90]], dtype=uint8)
 
-              [[ 2.],
-                [ 5.]],
-
-              [[ 0.],
-                [ 2.]]])
+        >>> pe = PatchExtractor(patch_size=(1024, 2), random_state=1234)
+        >>> pe.fit(X)
+        >>> X_transform = pe.transform(X)
+        >>> X_transform[0]
```

Comment:
maybe just show the shape of X_transform? And maybe use a different patch size? this seems strange.